Description: Fix man pages typos
 Fix typos in the slurmdbd.conf.5 and slurm.conf.5 man pages
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>
Last-Update: 2018-02-24

--- a/doc/man/man5/slurm.conf.5
+++ b/doc/man/man5/slurm.conf.5
@@ -3578,7 +3578,7 @@
 Specifies the nodes which are to not be placed in power save mode, even
 if the node remains idle for an extended period of time.
 Use Slurm's hostlist expression to identify nodes with an optional ":" separator
-and count of nodes to exclude from the preceeding range.
+and count of nodes to exclude from the preceding range.
 For example "nid[10-20]:4" will prevent 4 usable nodes (i.e IDLE and not 
 DOWN, DRAINING or already powered down) in the set "nid[10-20]" from being
 powered down.
